content/pt-br/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Comunicação entre contêineres no mesmo pod usando um volume compartilhado
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+Esta página mostra como usar um Volume para se comunicar entre dois Containers rodando
+no mesmo pod. Veja também como permitir que os processos se comuniquem por
+[compartilhamento do processo de namespace](/docs/tasks/configure-pod-container/share-process-namespace/)
+entre os containers.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Criando um pod que executa dois contêineres
+
+Neste exercício, você cria um pod que executa dois contêineres. Os dois recipientes
```

Comment:
Some bits here:
* Pod should be rendered with a capital P.
* I'm confused as to why you used "contêineres" in the first sentence, but then used "recipiente". It should be "contêineres" in the second sentence as well.